Automattic/jetpack
# Jetpack Forms: Update the styling on response emails

Jetpack Forms sends out lots of emails to website owners and they are very basic right now. This PR will add filters to change content more easily and a new, more attractive default style to those emails.

Filters added:
* jetpack_forms_response_email - fired when creating the HTML of the response. An array is passed to it of response field label => value arrays and return a string with formatted HTML if it is to be used.
* jetpack_forms_response_email_title - The title of the email message shown in bold in the email body. Default: "You got a new response!"
* jetpack_forms_response_email_footer - an array of meta information about the response such as time, IP, source URL, and user information.
* jetpack_forms_respone_email_template - fired when wrapping the message in HTML tags. Pass it a string containing the boiler plate open and closing tags of a HTML page with %1$s for the message title, %2$s for the response, %3$s for the response link, %4$s for the link to the form on the website, %5$s for the meta data footer.

This allows website owners to customize the response emails sent from their website.

For context: pcsBup-yo-p2

Emails before:
![image](https://user-images.githubusercontent.com/5120357/236024812-ddceec6f-a788-4ae1-932a-f2853d9f5163.png)

After:

<img src="https://user-images.githubusercontent.com/5120357/236025358-14a5e1e6-244b-4964-b6a9-0a61587e5007.png" width="200px" height="644px">

## Does this pull request change what data or activity we track or use?
No

### Other information:


## Jetpack product discussion
pcsBup-yo-p2

## Testing instructions:
Apply patch
Add a contact form to a website and fill it in
Check your inbox.
The email will look much the same as before but you'll see two extra parts:
* A header "You got a new response!" message
* A footer "View Response" message
